stories/migrazione.stories.mdx

```diff
@@ -0,0 +1,90 @@
+import { Meta } from '@storybook/addon-docs';
```

Comment:
I think this content does not belong here.

This is a migration guide from Bootstrap Italia v.1 to Bootstrap Italia v.2 which is already present on the BI documentation page. I think the React kit should propose its own migration guide, a guide which would talk about components and skip all these implementation details which belongs to the a peer dependency.

The actual migration guide from v4 to v5 should be included in the `changelog-page`, as already presented for v3 to v4.

Reply:
removed file in #937